pages/projects.js

```diff
@@ -80,7 +82,7 @@ export default function Projects() {
     );
   } else {
     if (currentReview) {
-      return <Review submission={currentReview} />;
+      return <Review submission={currentReview} start_time={startTime} />;
```

Comment:
why do you use `start_time` here? for consistency, I'd rather use `startTime={startTime}` which should be perfectly fine. Curious to hear your rationale for switching between one and the other?